docs/UserGuide.md

```diff
@@ -629,34 +629,27 @@ Format: `list`
 
 ### Finding patients: `find`
 
-Finds patients whose names contain any of the given keywords.
-
-Format: `find [KEYWORD]… [n/NAME]… [p/PHONE]… [e/EMAIL]… [a/ADDRESS]… [t/TAG]… [c/CONDITION]… [d/TASK_DESCRIPTION]… [m/MEDICATION]… [r/REMARK]…`
+Finds patients whose parameters contain the given keywords.
 
+Format: `find [KEYWORD]… [xyz/XYZ_KEYWORD]…`
```

Comment:
Another change I made in the reformatted UG was to bold the command words in every "Format" line.
```suggestion
Format: **`find`**`[KEYWORD]… [xyz/XYZ_KEYWORD]…`
```